pl_examples/basic_examples/profiler_example.py

```diff
@@ -0,0 +1,56 @@
+import torch
+import torchvision
+import torchvision.models as models
+import torchvision.transforms as T
+
+from pytorch_lightning import LightningDataModule, LightningModule, Trainer
+"""
+This will generate 2 traces: one for `training_step` and one for `validation_step`.
+Open Chrome and copy/paste this url: `chrome://tracing/`.
```

Comment:
@tchaton Please add the steps in readme that also describe how to load and visualize the traces in the new PyTorch Tensorboard plugin https://github.com/pytorch/kineto/tree/master/tb_plugin

Reply:
Done !